test/access-control/int.spec.ts

```diff
@@ -522,6 +524,44 @@ describe('Access Control', () => {
 
       expect(docs).toHaveLength(1)
     })
+
+    it('should ignore false access on query constraint added by top collection level access control', async () => {
+      await payload.create({
+        collection: 'fields-and-top-access',
+        data: { secret: '123456' },
+      })
+      const { id } = await payload.create({
+        collection: 'fields-and-top-access',
+        data: { secret: '12345' },
+      })
+      await payload.create({
+        collection: 'fields-and-top-access',
+        data: { secret: '123456' },
+      })
+
+      const resFind = await payload.find({
+        overrideAccess: false,
+        collection: 'fields-and-top-access',
+      })
+      expect(resFind.docs[0].id).toBe(id)
+
+      const resFindDraft = await payload.find({
+        draft: true,
+        overrideAccess: false,
+        collection: 'fields-and-top-access',
+      })
+
+      expect(resFind.docs[0].id).toBe(id)
+      expect(resFindDraft.docs[0].id).toBe(id)
+
+      const res = await payload.findByID({
+        id,
+        collection: 'fields-and-top-access',
+        overrideAccess: false,
+      })
+
+      expect(res).toBeTruthy()
+    })
```

Comment:
Can you add a comment on what this additional assertion is for?
I assume you're testing the same access control functionality for `findByID`.